src/matlab2tikz.m

```diff
@@ -1613,18 +1613,15 @@ function assertRegularAxes(handle)
 % not too fancy plots.
     str = '';
 
-    if ~isVisible(h)
-        return
-    end
-
```

Comment:
It's hardly put in the end. Personally, I rather have 1 "if" per concept instead of first checking visibility and afterwards do a more thorough check on visibility (by means of markers and so on).